README.md

```diff
@@ -67,14 +67,32 @@ Regarding continuous integration, the `generate.sh` script will create:
 depending on whether `meta.yml` contains `travis: true` or `action: true`
 or `circleci: true`.
 
-If you only want to generate certain files, please specify them as shell script arguments:
+For `coq-community` projects, using `travis: true` is currently
+disabled due to a very limed build-time allowance from Travis. Using
+GitHub actions is a reasonable choice. 
+
+If you only want to generate certain files, you can specify them as arguments to the shell script:
 ```shell
-$TMP/generate.sh README.md coq-aac-tactics.opam
+$TMP/generate.sh README.md coq-action.yml
+```
+
+As an alternative to using `generate.sh`, the configuration files can
+also be generated by calling `mustache` directly.  Asuming the
+templates are available in a sibling directory, the configuration and
+boilerplate files for [AAC_tactics](https://github.com/coq-community/aac-tactics) are
+generated as follows:
+
+```shell
+mustache meta.yml ../templates/README.md.mustache > README.md
+mustache meta.yml ../templates/coq.opam.mustache > coq-aac-tactics.opam
+mustache meta.yml ../templates/default.nix.mustache > default.nix
+mustache meta.yml ../templates/.travis.yml.mustache > .travis.yml
```

Comment:
Maybe reduce this example to just the `README`? This is not a very frequent use case anyway, and the list of file is already out-of-date (Travis).

Reply:
@Zimmi48 done! I also added some headlines to structure the text and move this alternative out of the main flow.